Playground/CMakeLists.txt

```diff
@@ -1,20 +1,17 @@
 set(exe_name Playground)
-########################################
+###############################################
 # External Libraries
 xpGetExtern(externIncs externLibs PUBLIC boost)
-########################################
+###############################################
 set(${exe_name}_srcs
   main.cpp
   )
-
 source_group("" FILES ${${exe_name}_srcs})
-
-
 add_executable(${exe_name} ${${exe_name}_srcs})
+###############################################
 MESSAGE(STATUS "boost: " ${externLibs})
```

Comment:
This message outputs this list (without newlines):
```
Boost::atomic
Boost::chrono
Boost::container
Boost::date_time
Boost::exception
Boost::filesystem
Boost::graph
Boost::iostreams
Boost::json
Boost::log
Boost::log_setup
Boost::nowide
Boost::prg_exec_monitor
Boost::program_options
Boost::random
Boost::regex
Boost::serialization
Boost::system
Boost::test_exec_monitor
Boost::thread
Boost::timer
Boost::unit_test_framework
Boost::wserialization
```

Reply:
That list is identical to the boost directories in here
![image](https://user-images.githubusercontent.com/14927087/168429830-2e66f91b-b3e4-460b-9785-235fc8788900.png)

Reply:
I'm pretty sure I just don't know how to properly include asio or beast in `target_link_libraries`

Reply:
Wait a second, are asio and beast automatically linked? I was able to include them, initialize `io_context` and `tcp_stream`, then build without any errors in `SolutionNameMain.cpp`
```cpp
#include <iostream>
#include <string>

#include <boost/asio.hpp>
#include <boost/beast.hpp>

int main(int argc, char* argv[])
{
  std::cout << "This is main" << std::endl;

  boost::asio::io_context ctx;
  boost::beast::tcp_stream stream(ctx);

  return 0;
}
```